x/auth/keeper/keeper.go

```diff
@@ -233,3 +238,30 @@ func (ak AccountKeeper) UnmarshalAccount(bz []byte) (types.AccountI, error) {
 
 // GetCodec return codec.Codec object used by the keeper
 func (ak AccountKeeper) GetCodec() codec.BinaryCodec { return ak.cdc }
+
+// add getter for bech32Prefix
+func (ak AccountKeeper) GetBech32Prefix() string { return ak.bech32Prefix }
+
+// ConvertAddressStringToBytes encodes text to bytes
+func (ak *AccountKeeper) ConvertAddressStringToBytes(text string) ([]byte, error) {
+	_, bz, err := bech32.DecodeAndConvert(text)
+	if err != nil {
+		return nil, err
+	}
+
+	if err := sdk.VerifyAddressFormat(bz); err != nil {
+		return nil, err
+	}
```

Comment:
This function uses the global Config under the hood, so we should avoid it.

We can deprecate this function, and create a new pure one in `address.VerifyFormat()`